pkg/server/filters/nodechanges.go

```diff
@@ -56,7 +56,7 @@ func WithNodeChanges(h http.Handler, uncachedLocalClient, uncachedVirtualClient
 						return
 					}
 
-					responsewriters.WriteObjectNegotiated(s, negotiation.DefaultEndpointRestrictions, corev1.SchemeGroupVersion, w, req, http.StatusOK, updatedNode)
+					responsewriters.WriteObjectNegotiated(s, negotiation.DefaultEndpointRestrictions, corev1.SchemeGroupVersion, w, req, http.StatusOK, updatedNode, false)
```

Comment:
**responsewriters.WriteObjectNegotiated** now accepts an extra argument _listGVKInContentType_ that controls whether the object is serialized with GVK info.
https://github.com/kubernetes/apiserver/blob/ea4e287db4aaada47d30b9d7b18f831a9729a93e/pkg/endpoints/handlers/responsewriters/writers.go#L289